### PR TITLE
Moar improvements

### PR DESCRIPTION
**Pull request by vagkos1**

### CODE REVIEW

**Comment by vagkos1** [comment]
more to come, wait for it!

File: 3-tutorial.md [hunk 1]
```diff
@@ -622,7 +622,7 @@ You may notice the *Endpoint*, just like *Deployments* create *ReplicaSets* whic
 will differ).
 
 ```bash
-❯ kubectl get endpoint kuard -o yaml
+❯ kubectl get endpoints kuard -o yaml
```
Comment (REBELinBLUE) [inline 3-tutorial.md]: can you change to `endpoints/kuard` since that is the syntax used in the rest of the guide

**Comment by REBELinBLUE** [comment]
Conflicts now unfortunately

**Comment by vagkos1** [comment]
close in favor of https://github.com/REBELinBLUE/kube-guide/pull/4

**Pull request closed without merging**